index.html

```diff
@@ -9,24 +9,57 @@
   </head>
   <body>
     <div class="container" id="container">
-      <h1>Awesome Books</h1>
+      <ul class="nav-list">
+        <li class="navl"><a id="contact-click" href="#home">Contact</a></li>
+        <li><a href="#news" id="add-new">Add New</a></li>
+        <li><a href="#contact" id="list">List</a></li>
+        <li id="awesome-logo"><a href="#about">Awesome Book</a></li>
+      </ul>
```

Comment:
- [ ] To enhance the user experience of this design kindly ensure that an active menu link is highlighted(color change or change in opacity). In other words, the current active page should have the corresponding menu item highlighted, as can be seen in the provided [wireframe design](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website.md). Kindly ensure to improve upon this aspect in your design, as we really want our designs to as much as possible mimic the provided wireframe template.

**Your design menu:**
![image](https://user-images.githubusercontent.com/12745474/144444721-4234a9d8-e8af-4fb3-8675-994f1c278fac.png)


**The wireframe menu design:**
![image](https://user-images.githubusercontent.com/12745474/129183812-a30acaa9-4f6c-4d4e-973b-5dd83a591c8d.png)